### DISCUSSION TITLE
Using Directus with CapacitorJS

### DISCUSSION THREAD

**Discussion by halwesit** [directus/directus#9878] [Archive]
If you familiar with the [CapacitorJS](https://capacitorjs.com/) , you may know that CapacitorJS can convert your webapp to mobile apps, 
For using any webapp with CapacitorJS, we have to build its packages and we should have a `dist` folder with and `index.html` to use it with CapacitorJS, but I didn't found any build command in Directus. 

So I want to know its possible to use CapacitorJS with the Directus?

**Comment by rijkvanzanten** [comment]
You wouldn't build directus itself into your app in order to use it. Instead, the app you'd build using capacitorJS would use the rest/graphql api to connect the data 👍🏻

**Comment by rijkvanzanten** [comment]
You wouldn't build directus itself into your app in order to use it. Instead, the app you'd build using capacitorJS would use the rest/graphql api to connect the data 👍🏻
